2024Robot_Team3484/src/main/cpp/commands/LauncherCommand.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "commands/LauncherCommand.h"
+
+using namespace LauncherConstants;
+
+LauncherCommand::LauncherCommand(LauncherSubsystem* Launcher_subsytem) 
+: _Launcher{Launcher_subsytem}{
+    AddRequirements(_Launcher);
+}
+void LauncherCommand::Initialize(){
+    _Launching = false;
+    _Launcher->setLauncherRPM(RPM);
+}
+void LauncherCommand::Execute(){
+    if(_Launcher){ //here is where we would have our run intake
+
+    }
+    else{
+        if(_Launcher->atTargetRPM()){//when intake is done replace with if(_Launcher->atPoint() and _intake->finshed angle()(**replace filler names with real ones**))   
+            _Launching = true;
+        }
+    }
```

Comment:
While this is clearly not complete, I'm not sure what the flow of logic here is. The `if` statement is maybe checking if the launcher instance is not 0 (this may work as a null check, or may crash the robot with a null pointer exception). If this command is waiting on other branches, I recommend writing out in a comment what the expected flow of logic should look like.